solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/FlowReceiverContainer.java

```diff
@@ -153,7 +173,7 @@ public MessageContainer receive(Integer timeoutInMillis) throws JCSMPException,
 					realTimeout = 0;
 				}
 			} catch (ArithmeticException e) {
-				logger.debug("Failed to compute real timeout", e);
+				LOGGER.debug("Failed to compute real timeout", e);
```

Comment:
// for later clean up/removal of realTimeout & expiry as they not needed any more